contracts/BatchReveal.sol

```diff
@@ -350,4 +460,16 @@ abstract contract BatchReveal is
         hasBeenForceRevealed = true;
         emit Reveal(batchNumber, batchToSeed[batchNumber]);
     }
+
+    /// @dev Determines if batch reveal is needed.
+    /// We assume batch reveal is needed if the configuration has
+    /// not been initialized or if revealBatchSize is not 0.
+    function isBatchReveal() internal view returns (bool) {
```

Comment:
Why can we use a modifier for checking this?

```solidity
modifier batchRevealEnabled() {
    if (revealBatchSize != 0) {
        revert Launchpeg__RevealNextBatchNotAvailable();
    }
    _;
}
```

Reply:
ok, will do